build.gradle

```diff
@@ -404,8 +404,7 @@ allprojects {
 
     applyKotlinConfig = { ->
       compileKotlin {
-        // TODO (#785): revert this setting
```

Comment:
Pls update intellijVersions.properties and libs.versions.toml for Kotlin plugin (still needs to be @-pinned, at higher version tho)

Reply:
(`s/.toInt()/.code` probably once Kotlin is bumped)